Documentation/op-guide/configuration.md

```diff
@@ -201,6 +201,11 @@ The security flags help to [build a secure etcd cluster][security].
 + default: none
 + env variable: ETCD_TRUSTED_CA_FILE
 
+### --auto-tls
```

Comment:
should this be `--client-auto-tls` to be consistent with the `--peer-auto-tls` option?